src/archan/plugins/providers.py

```diff
@@ -23,14 +23,14 @@ class CSVInput(Provider):
         Argument("categories_delimiter", str, "If set, used as delimiter for categories."),
     )
 
-    def get_data(self, file_path=sys.stdin, delimiter=",", categories_delimiter=None):
+    def get_data(self, file_path=None, delimiter=",", categories_delimiter=None):
         """
         Implement get_dsm method from Provider class.
 
         Parse CSV to return an instance of DSM.
 
         Args:
-            file_path (str/fd): path or file descriptor.
+            file_path (str/fd): path or file descriptor. If None, uses sys.stdin.
```

Comment:
On second thought, this might be better off as a `Optional[str]` type. File descriptors don't appear to be supported and if they were, they would also create issues with deepcopy

Reply:
Sure, feel free to remove `/fd`!